server/config/routes.js

```diff
@@ -741,4 +743,11 @@ exports.configure = function configure(app) {
 
   // unit
   app.get('/unit/:roleUuid', unitCtrl.list);
+
+  // department
+  app.get('/department', department.read);
```

Comment:
It looks like most of our URLs are plural - let's name these `/departments` for consistency.